src/foremast/consts.py

```diff
@@ -197,10 +197,15 @@ def _generate_security_groups(config_key):
 DOMAIN = validate_key_values(CONFIG, 'base', 'domain', default='example.com')
 ENVS = set(validate_key_values(CONFIG, 'base', 'envs', default='').split(','))
 REGIONS = set(validate_key_values(CONFIG, 'base', 'regions', default='').split(','))
+MANUAL_PIPELINE_SYNONYMS = set(
+    validate_key_values(CONFIG, 'base', 'manual_pipeline_synonyms', default='manual').split(','))
 ALLOWED_TYPES = set(
-    validate_key_values(CONFIG, 'base', 'types', default='ec2,lambda,s3,datapipeline,rolling,manual').split(','))
+    validate_key_values(CONFIG, 'base', 'types', default='ec2,lambda,s3,datapipeline,rolling,manual')
+    .split(',')) | MANUAL_PIPELINE_SYNONYMS
 RUNWAY_BASE_PATH = validate_key_values(CONFIG, 'base', 'runway_base_path', default='runway')
 TEMPLATES_PATH = validate_key_values(CONFIG, 'base', 'templates_path')
+TEMPLATES_SCHEME_IDENTIFIER = validate_key_values(CONFIG, 'base', 'templates_scheme_identifier', default='templates://')
+TEMPLATE_VARIABLES_KEY = validate_key_values(CONFIG, 'base', 'template_variables_key', default="template_variables")
```

Comment:
Does this need to be configurable or could it just be a common template key? We can add it to the docs and that can help remove some of the logic checks you are doing as well?